materialize-redshift/driver.go

```diff
@@ -662,14 +687,6 @@ func (d *transactor) commit(ctx context.Context, fenceUpdate string, hasUpdates
 	}
 	defer conn.Close(ctx)
 
-	// Truncate strings within SUPER types by setting this option, since these have the same limits
```

Comment:
See note above about not needing this truncation anymore, since it is done by the runtime automatically now.